config/F16.config

```diff
@@ -4,6 +4,17 @@ process {
         memory = 1.GB
         }
 
+    withName: preprocess_BAM_GRIDSS {
+        cpus = 4
+        memory = 10.GB
+        retry_strategy {
+            memory {
+                strategy = 'exponential'
+                operand = 2
+                }
+            }
+        }
```

Comment:
**comment**: Any reason to only add this for F16 and not any of the other partition configs?